src/fun/transform/desugar_match_defs.rs

```diff
@@ -301,8 +302,10 @@ fn num_rule(
       let mut body = rule.body.clone();
       if let Some(var) = var {
         let last_num = *nums.last().unwrap();
-        let var_recovered = Term::add_num(Term::Var { nam: pred_var.clone() }, Num::U24(1 + last_num));
+        let curr_num = 1 + last_num;
```

Comment:
```suggestion
        let cur_num = 1 + last_num;
```
Let's go with `cur` to mean "current" everywhere in the repo. (this is the only place that uses `curr`)